ArduPlane/soaring.cpp

```diff
@@ -26,8 +26,8 @@ void Plane::update_soaring() {
     case Mode::Number::CRUISE:
         g2.soaring_controller.suppress_throttle();
         break;
-    case Mode::Number::LOITER:
-        // Never use throttle in LOITER with soaring active.
+    case Mode::Number::THERMAL:
```

Comment:
but what happens if we're in Loiter? It just doesn't work now?

Reply:
THERMAL is now what LOITER was with soaring enabled.
LOITER is now what LOITER was without soaring enabled.